src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -266,6 +266,9 @@ const processDataPoints = (graphConfig, dataTarget) => {
     // Update the interpolation type
     dataTarget.interpolationType = getInterpolationType(dataTarget.type);
 
+    dataTarget.strokeDasharray =
+        (dataTarget.style && dataTarget.style.strokeDasharray) || "0";
```

Comment:
This is good for now but can we make this more maintainable? 
How about: 
```js
// Since we are using the stroke-dasharray property everytime in line 327, let's have a helper function
const getStrokeDashArray = (style) => getDefaultValue(style.strokeDashArray, "0");

// Provide the function in the line 269 like...
dataTarget.style = getDefaultValue(dataTarget.style, "{}");
dataTarget.style = {
    strokeDashArray: getStrokeDashArray(dataTarget.style)
}
```
This way any future additions to `style` can be treated in similar fashion.

Reply:
Updated